api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionEditPaths.scala

```diff
@@ -66,36 +67,56 @@ trait SubmissionEditPaths
   def submissionSingleEditPath(institutionId: String) =
     path("filings" / Segment / "submissions" / IntNumber / "edits" / Segment) { (period, seqNr, editType) =>
```

Comment:
What would you think about splitting this endpoint definition into two separate ones? Just a question to consider. Also, if this is out of scope for this ticket, we could make it a separate PR.

It seems like we could have one for the Macro edits `GET` and `POST`
```scala
     path("filings" / Segment / "submissions" / IntNumber / "edits" / "macro" ) { (period, seqNr) =>
```
and one for the remaining edit types `GET`

```scala
     path("filings" / Segment / "submissions" / IntNumber / "edits" / Segment) { (period, seqNr, editType) =>
```